src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerListener.cs

```diff
@@ -54,6 +75,14 @@ internal class CosmosDBTriggerListener : IListener, Microsoft.Azure.Documents.Ch
             this._healthMonitor = new CosmosDBTriggerHealthMonitor(this._logger);
         }
 
+        public string Id
+        {
+            get
+            {
+                return $"CosmosDBTrigger-{_hostName}-{_monitorCollection.DatabaseName}-{_monitorCollection.CollectionName}".ToLower();
```

Comment:
Please follow the same pattern as in QueueListener for this - e.g. `return $"{_functionId}-QueueTrigger-{_queue.Name}".ToLower();`

Reply:
I should have mentioned I had questions about how to replicate that queue scheme. I'm not sure what the equivalent of queue name would be for Cosmos. I've replaced `_queue.name` with `monitoredDb-monitoredCollection` if that works.

Reply:
Yes, that's it